docs/windows.md

```diff
@@ -76,6 +78,33 @@ You will be asked to restart your machine, do so, and then continue with the res
     time="2024-02-26T10:42:16+03:00" level=warning msg="currently, only the default worker can be used."
     time="2024-02-26T10:42:16+03:00" level=info msg="running server on //./pipe/buildkitd"
     ```
+
+    **Running `buildkitd` with the CNI:**
+
+    Note that the above simple run will not have the networking bit setup;
+    for instance you won't be able to access the internet from the builds e.g. downloading resources.
+
+    Follow the instructions in the [CNI / Networking Setup](#cni--networking-setup) section.
+    Once that is done, you can now start `buildkitd` providing the binary and config paths to the flags:
+
+    ```powershell
+    buildkitd `
+        --containerd-cni-config-path="C:\Program Files\containerd\cni\conf\0-containerd-nat.conf" `
+        --containerd-cni-binary-dir="C:\Program Files\containerd\cni\bin"
+    ```
+
+    You can also run `buildkitd` as a _Windows Service_:
+
+    ```powershell
+    buildkitd `
+        --register-service `
+        --service-name buildkitd `
+        --containerd-cni-config-path="C:\Program Files\containerd\cni\conf\0-containerd-nat.conf" `
+        --containerd-cni-binary-dir="C:\Program Files\containerd\cni\bin" `
+        --debug `
+        --log-file="C:\Windows\Temp\buildkitd.log
+    ```
+
```

Comment:
You can also add something like:


Buildkitd on Windows depends on `containerd`. For `buildkitd` to work, you will also need `containerd` present and running on your system.

Assuming that the `containerd` service name is called "containerd", we can make the `buildkitd` service dependent on `containerd`:

```powershell
sc.exe config buildkitd depend= containerd
```

And we can set the service to start automatically:

```powershell
Set-Service -StartupType Automatic buildkitd
```

Reply:
will do, thanks!